bobtemplates/plone/view.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""Generate view."""
+
+from bobtemplates.plone.base import base_prepare_renderer
+from bobtemplates.plone.base import git_commit
+from bobtemplates.plone.base import update_file
+from lxml import etree
+from mrbob.bobexceptions import SkipQuestion
+from mrbob.bobexceptions import ValidationError
+
+import os
+import stringcase
+
+
+def get_view_name(configurator, question):
+    """Generate view default name from python class"""
+    if configurator.variables['view_python_class']:
+        class_name = configurator.variables['view_python_class_name']
+        view_name = stringcase.snakecase(class_name).replace('_', '-')
+        question.default = view_name
+    else:
+        question.default = 'my-view'
+
+
+def get_template_name(configurator, question):
+    if configurator.variables['view_template']:
```

Comment:
this variable name is confusing me. view_name doesn't say anything.
Please go over the names and name them, so that they tell me for what they actually are.